src/components/common/Extensions/UserHub/partials/StakesTab/useStakesByFilterType.tsx

```diff
@@ -42,13 +46,28 @@ export const useStakesByFilterType = () => {
   const { motionStatesMap, loading: motionStatesLoading } =
     useNetworkMotionStates(motionIds);
 
-  const stakesWithStatus = useMemo(
+  const { nativeToken } = colony || {};
+
+  const stakesWithStatus: StakesTabContentListItem[] = useMemo(
     () =>
       userStakes.map((stake) => ({
-        ...stake,
+        key: stake.id,
+        // @TODO: Replace with action custom title
+        title: stake.action?.type || '',
+        date: intl.formatDate(stake.createdAt),
+        stake: (
+          <Numeral
+            value={stake.amount}
+            decimals={getTokenDecimalsWithFallback(nativeToken?.decimals)}
+            suffix={nativeToken?.symbol}
+          />
+        ),
```

Comment:
These are all view implementation details and they shouldn't be here. Please could you restore the original logic of this hook?